openshift/templates/nginx-runtime/nginx.conf.template

```diff
@@ -1,3 +1,5 @@
+# This nginx server config now lives in ConfigMaps (nginx-conf) in the dev, test, prod projects respectively
```

Comment:
Sounds good! Thanks for maintaining this copy; good to know we're using a more flexible system now